sondage/rediger-taches.md

```diff
@@ -0,0 +1,187 @@
+        <h2 class="mrgn-tp-0 text-success h4"><span class="glyphicon glyphicon-ok-circle"></span> Quoi faire </h2>
+        <p>Recherche [type spécifique] de dossiers</p>
+    </section>
+    <section class="col-md-5 brdr-lft col-xs-6">
+        <h2 class="mrgn-tp-0 text-danger h4"><span class="glyphicon glyphicon-remove-circle"></span> Quoi éviter </h2>
+        <p>Recherche de dossiers </p>
+    </section>
+</div>
+
+* * *
+
+### Chargement frontal avec des mots clés sur la tâche ou le service
+
+#### Services et produits
+
+* Chercher des appels d’offres pour des marchés de l’État
+* Constituer une entreprise
+* Carte de résident permanent (obtenir, renouveler, remplacer)
+* Prestation canadienne de la relance économique pour proches aidants (PCREPA)
+
+#### Services (où plusieurs tâches sont incluses au même niveau)
+
+* AE (assurance-emploi) — vérifier l’admissibilité
+* AE (assurance-emploi) — faire une demande
+* AE (assurance-emploi) — soumettre le formulaire de déclaration
+
+* * *
+
+### Utiliser une formulation normalisée pour les tâches communes
+
+<div class="row wb-eqht mrgn-tp-lg">
+    <section class="col-md-12 col-xs-12">
+        <h2 class="mrgn-tp-0 text-success h4"><span class="glyphicon glyphicon-ok-circle"></span> Quoi faire</h2>
+        <ul>
+            <li> Emplois avec [x]/Emplois étudiants avec [x]</li>
+            <li> Communiquez avec [nom de l’institution] pour obtenir de l’aide</li>
+            <li> [Nom] compte (inscription, connexion, aide)</li>
+            <li> Vérifiez votre statut [nom]/vérifiez le statut de [nom]</li>
+            <li> Demander un [programme/fonds/avantage]</li>
+        </ul>
+    </section>
+</div>
+
+* * *
+
+### Optez pour les termes orientés vers la navigation plutôt que pour le terme lié aux capacités «&nbsp;voir, consulter&nbsp;»
+
+<div class="row wb-eqht mrgn-tp-lg">
+    <section class="col-md-5 col-xs-6">
+        <h2 class="mrgn-tp-0 text-success h4"><span class="glyphicon glyphicon-ok-circle"></span> Quoi faire </h2>
+        <p>	Obtenir mon T4 ou Vérifier mon T4 </p>
+        <p> Obtenir mon relevé d’emploi ou Vérifier mon relevé d’emploi </p>
+    </section>
+    <section class="col-md-5 brdr-lft col-xs-6">
+        <h2 class="mrgn-tp-0 text-danger h4"><span class="glyphicon glyphicon-remove-circle"></span> Quoi éviter </h2>
+        <p>Voir mon T4 </p>
+        <p>Voir mon relevé d’emploi </p>
+    </section>
+</div>
+
+* * *
+
+### Évitez les mots complexes lorsqu’il y a une solution de rechange plus simple
+
+<div class="row wb-eqht mrgn-tp-lg">
+    <section class="col-md-5 col-xs-6">
+        <h2 class="mrgn-tp-0 text-success h4"><span class="glyphicon glyphicon-ok-circle"></span> Quoi faire </h2>
+        <p>Lire le [nom]</p>
+    </section>
+    <section class="col-md-5 brdr-lft col-xs-6">
+        <h2 class="mrgn-tp-0 text-danger h4"><span class="glyphicon glyphicon-remove-circle"></span> Quoi éviter </h2>
+        <p>Consultez le [nom]</p>
```

Comment:
This only applies to english...

Reply:
Same comment as in the English equivalent. Issue should be open to figure out how to avoid having missing content even when the situation is specific to one language.